index.html

```diff
@@ -47,149 +46,179 @@ <h2 class="desktop">Hi there. I'm Alfred</h2>
-                </nav>
+                </div>
             </div>
+            <!-- Framework Section -->
+            <div class="top-card-one">
+                <div class="fit">
+                    <img src="images/icon-frameworks2.svg" alt="" class="ml">
+                    <h2>Frameworks</h2>
+                    <ul class="col-one col-two">
+                        <li class="col"><a href="">Bootstrap</a></li>
+                        <li class="col"><a href="">Ruby on rails</a></li>
+                        <li class="col"><a href=""> RSpec </a></li>
+                        <li class="col"><a href=""> capybara </a></li>
+                        <li class="col"><a href=""> selinium </a></li>
+                    </ul>
+                </div>
+            </div>
+            <!-- electricity -->
+            <div class="top-card-one">
+                <div class="fit">
+                    <img src="images/icon-skills3.svg" alt="" class="ml">
+                    <h2>Skills</h2>
+                    <nav>
+                        <ul class="col-one col-two">
+                            <li class="col"><a href="">Codekit</a></li>
+                            <li class="col"><a href="">GItHUb</a></li>
+                            <li class="col"><a href=""> Codepen </a></li>
+                            <li class="col"><a href=""> Gitlab </a></li>
+                            <li class="col"><a href=""> Terminal </a></li>
+                        </ul>
+                    </nav>
+                </div>
+            </div>
+            </section>
+        <section class="its-time-to-flex">
             <p class="p3 sp">I'm always interested in hearing about new projects,
                 so if you'd like to chat please get in touch.</p>
             <!-- contact form -->
-            <form action="#" class="forms">
-                <input type="text" id="fname" name="fname" class="input_one" placeholder="Full name"><br>
-                <input type="email" id="lname" name="email" class="input_one" placeholder="Email address"><br><br>
-                <textarea id="talk" name="talk" class="message_box" placeholder="Enter text here here"></textarea>
-                <input type="submit" class="button" value="Get in touch">
+            <form  action="https://formspree.io/f/mknypyqz" method="POST" class="forms">
+                <input type="text" id="fname" name="name" class="input_one" placeholder="First name" required maxlength="30">
+                <input type="text" id="lname" name="name" class="input_one last-input desktop" placeholder="Last name" required maxlength="30">
+                <input type="email" name="_replyto" class="input_two" placeholder="Email address" required><br><br>
+                <textarea id="message" name="message" class="message_box" placeholder="Enter text here here" required maxlength="500"></textarea>
+                <input type="submit" class="button small-button" value="Get in touch">
```

Comment:
- [ ] Please your submit button is not working. Kindly fix that and make sure the form is submitted successfully.